### ISSUE TITLE
Emoji picker

### ISSUE THREAD

**Issue by glebm** [thredded/thredded#225]
Hmm... https://github.com/ID25/rails_emoji_picker

**Comment by jayroh** [comment]
I could go either way on this. I've bookmarked the same library a few weeks ago so I could refer to it later and have been thinking about this as well.

On the one hand, this would be cool. It'd be nice.

On the other, this is something that could rather easily be implemented by whoever decided to use thredded in/with their app. Therefore I don't think it's the responsibility of thredded to maintain this functionality. Similarly - file uploads, keyboard shortcuts, etc.

Still open for discussion though :)

**Comment by zoopyserg** [comment]
Yeah well see out of the box it messes up thredded's front end setup.
So it would be great if it just had it, you know.  
With all JS aspects fixed. 
Cause otherwise I do `rake thredded:install:emoji` and think "okay where's my emoticons. And then I see "oh snap I gotta do this and that, and duplicate the thredded's view partial, and use that gem, and see why my input is no longer white" etc. 

If the concern is that not everybody needs emoticons - then there could be just another line in config initializer which would say "use_emoticons_in_messageboard_description = true/false" & "use_emoticons_in_post_message_field = true/false". That's it.
I mean it's great anyway, but this thing would be useful.

**Comment by glebm** [comment]
@zoopyserg gemoji dependency has been removed from Thredded, so this plugin should be feasible now.
